react/components/SearchBar/index.js

```diff
@@ -82,13 +56,13 @@ class SearchBarContainer extends Component {
 
     return (
       <SearchBar
+        // eslint-disable-next-line jsx-a11y/no-autofocus
         autoFocus={autoFocus}
         placeholder={placeholder}
         shouldSearch={shouldSearch}
```

Comment:
This will be `undefined`, since you removed it from the `state`, you can remove it here too

Reply:
Oh, forgot to remove it. Thx